roles/env_example

```diff
@@ -34,6 +34,17 @@ SHARES_LOGGER__CLICKHOUSE__MAX_RETRY_DELAY_SECS=1
 # Параметры каналов
 SHARES_LOGGER__PROCESSING__PRIMARY_CHANNEL_BUFFER_SIZE=10000
 SHARES_LOGGER__PROCESSING__BACKUP_CHECK_INTERVAL_SECS=1
+# Проверка блока
```

Comment:
Added environment variables for configuring block verification retries and delays.